# attacker msg 
# 投标时的cost值
float32 cost

# 无人机编号
int8 UAV_NUM

# 无人机状态
int8 status

# 无人机执行任务成功与否标识符
bool result
